src/webgpu/shader/validation/wgsl/duplicate-entry-point.fail.wgsl

```diff
@@ -1,12 +1,11 @@
 # v-0020 - Duplicate entry point <name, stage> pair.
```

Comment:
This isn't quite true any more. You can't have a duplicate entry point name at all anymore, regardless of the stage. So, we should add a second test with `stage(vertex)` and `stage(fragment)` with the same name to verify it fails.